cicecore/cicedynB/general/ice_init.F90

```diff
@@ -2372,6 +2377,32 @@ subroutine init_state
                         vicen, vsnon, &
                         ntrcr, trcrn)
 
+      ! set_state_var defines uvel/vel on U-Grid
```

Comment:
remove both of these comments -- the first one is not generally true, only for boxslotcyl, and the second one is redundant